test/deno.js

```diff
@@ -16,8 +16,6 @@ const fixtures = require('./mocha-fixtures.js')
 await fixtures.mochaGlobalSetup();
 
 const child_args = [
-  // args is required to be set manually, because there is currently no way to get all arguments from deno
-  '--allow-env', '--allow-read', '--allow-net', '--allow-run', '--allow-sys', '--allow-write',
```

Comment:
is this somehow not required anymore?

Reply:
Yeah this looks to cause issues in Deno 2.x